fluent-bundle/src/resolver/inline_expression.rs

```diff
@@ -147,11 +147,11 @@ impl<'p> WriteValue for ast::InlineExpression<&'p str> {
     }
 }
 
-impl<'p> ResolveValue for ast::InlineExpression<&'p str> {
-    fn resolve<'source, 'errors, R, M>(
-        &'source self,
-        scope: &mut Scope<'source, 'errors, R, M>,
-    ) -> FluentValue<'source>
+impl<'bundle> ResolveValue<'bundle> for ast::InlineExpression<&'bundle str> {
+    fn resolve<'ast, 'args, 'errors, R, M>(
+        &'ast self,
+        scope: &mut Scope<'bundle, 'ast, 'args, 'errors, R, M>,
+    ) -> FluentValue<'bundle>
```

Comment:
Praise: `'bundle` is much clearer than `'p` on why a lifetime exists, and how the reference is constrained.